packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropType from 'prop-types';
+import { CarouselInput, CarouselSlide, CarouselActions } from '@strapi/design-system/CarouselInput';
+import { IconButton } from '@strapi/design-system/IconButton';
+import { Box } from '@strapi/design-system/Box';
+import Plus from '@strapi/icons/Plus';
+
+const LogoInput = ({ customLogo, defaultLogo }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <CarouselInput
+      label={formatMessage({
+        id: 'Settings.application.customization.carousel.title',
+        defaultMessage: 'Logo',
+      })}
+      selectedSlide={0}
+      hint="Change the admin panel logo (Max dimension: 750*750, Max file size: TBC)"
+      previousLabel="Previous slide"
+      nextLabel="Next slide"
```

Comment:
These labels should be translated (hint, previousLabel, nextLabel)

Reply:
For the hint yes indeed - since the max file size is still to be determined I wondered if I should wait to add the translation, WDYT?
For the previous and next labels I can add them if you think it's best but they won't be used - we'll never have more than one slide - I think during a peer prog @soupette told me to not add them, if I remember well 🤔

Reply:
Indeed, it is not necessary